### ISSUE TITLE
Permission problems and lack of documentation around NFS

### ISSUE THREAD

**Issue by habitullence** [mitchellh/vagrant#5583]
Tried all the examples I could find scattered across the web on NFS export options but nothing changes my /etc/exports file:

1) Do these options exist?
2) Where are the docs?

To stop the permission problems would be great if Vagrant matched the host UID out-of-the-box.

**Comment by habitullence** [comment]
Seems to be a lot of talk out there about users with NFS permissions issues. Looks like NFS permissions issues would be fixed if you could change the vagrant uid to match host.

**Comment by habitullence** [comment]
To change the UID and fix the permissions problems I was having I tried to log on with root through Vagrant but this doesn't work (#5586). In the end I booted in VirtualBox change the UID of the Vagrant user with ```usermod -u 501 vagrant``` and now everything works OK.

I've had endless problems with Vagrant and NFS but once you understand a bit about NFS and figure a way round the chicken and egg problem of changing the UID smooth sailing with Vagrant NFS.

Would be great if Vagrant dealt with this out-of-the-box though.
